ci/includes/pinned-tools.sh

```diff
@@ -30,4 +30,4 @@ export LINTER="cleanupcode.exe"
 
 export MSBUILD="$(powershell  –ExecutionPolicy Bypass ./ci/find-msbuild.ps1)"
```

Comment:
Do we use this anymore?

Reply:
This and the associated script can be nuked, though it could happen in a follow-up PR once this is all stabilized.

Reply:
👍